### PR TITLE
Russian numbers (ordinal)

### PR DESCRIPTION
**Pull request by petr-tik**
read-string for ordinal russian digits fails, so removed it. 

Added more ordinal numbers and expanded for different genders (masculine, neuter, feminine) in different cases (accusative, dative, nominative)

### CODE REVIEW

File: resources/duckling/rules/ru.numbers.clj [hunk 1]
```diff
@@ -121,18 +121,11 @@
...
-              "семнадцатый" 17 "восемнадцатый" 18 "девятнадцатый" 19 "двадцатый" 20}
+   :value (get {"перв" 1 "втор" 2 "трет" 3 "четверт" 4 "пят" 5
+              "шест" 6 "седьм" 7 "восьм" 8 "девят" 9 "десят" 10 "одинадцат" 11
+              "двенадцат" 12 "тринадцат" 13 "четырнадцат" 14 "пятнадцат" 15 "шестнадцат" 16 "семнадцат" 17 "восемнадцат" 18 "девятнадцат" 19 "двадцат" 20}
               (-> %1 :groups first .toLowerCase))}
 
-  "ordinal (digits)"
-  #"0*(\d+)(-)?(ый|ой|ий)"
-  {:dim :ordinal
-   :value (read-string (first (:groups %1)))}  ; read-string not the safest
```
Comment (blandinw) [inline resources/duckling/rules/ru.numbers.clj]: can you explain why you removed this?

**Comment by blandinw** [comment]
Thank you for the contribution!
Let's find out why read-string was an issue and merge this 🇷🇺

File: resources/duckling/rules/ru.numbers.clj (hunk 1 above)
Comment (petr-tik) [inline resources/duckling/rules/ru.numbers.clj]: Have been fixing it at home. Will try to send a PR over the weekend with ordinal for digits and add more examples. I'm a native Russian speaker btw (the finnish surname can be confusing), if you are wondering where to proofread it.

**Comment by petr-tik** [comment]
Dealt with the read-string problem. No need for the 
(-)? 
matching group, Russian **very** seldom has hyphens in ordinal numbers (pre-Revolution Russian had some I think). 

Also, moved (**пят**надцат) ahead of (**пят**) to avoid premature regex evaluation

**Comment by petr-tik** [comment]
Just saw that you changed the dir structure. Does that mean you cannot accept my PR?

**Comment by patapizza** [comment]
Indeed, please rebase and let's merge it!

**Pull request closed without merging**